component/portal/src/main/java/org/exoplatform/portal/mop/rest/NavigationRest.java

```diff
@@ -180,10 +214,7 @@ private Response getNavigations(HttpServletRequest request,
 
   private static UserNodeFilterConfig getUserFilterConfig(Visibility[] visibilities) {
     UserNodeFilterConfig.Builder builder = UserNodeFilterConfig.builder();
-    builder.withReadWriteCheck()
-           .withVisibility(visibilities)
-           .withTemporalCheck()
-           .withReadCheck();
+    builder.withReadWriteCheck().withVisibility(visibilities).withTemporalCheck().withReadCheck();
```

Comment:
It's better to avoid changing code formatting for not changed parts

Reply:
Removed.